recipes/xdsmjs/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "xdsmjs" %}
+{% set version = "1.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xdsmjs-{{ version }}.tar.gz
+  sha256: 02450fe202d3fddf8408d893488ace76fea62c9a631019dd04b5978197be4ca8
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
```

Comment:
```suggestion
    - python >=3.6
```

Reply:
Yes but it works fine for python 2.7, this package is just a [dumb wrapper](https://github.com/OneraHub/XDSMjs/blob/master/py_xdsmjs/xdsmjs/__init__.py) with only an `__init__.py`.
I will add a lower bound though.